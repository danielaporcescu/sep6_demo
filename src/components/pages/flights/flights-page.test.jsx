import React from 'react';
import ReactDOM from 'react-dom';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import FlightsPage from './flights-page';

Enzyme.configure({ adapter: new Adapter() })

it('Renders correctly FlightsPage', () => {
    const wrapper = shallow(<FlightsPage />)
  
    expect(toJson(wrapper)).toMatchSnapshot();
  });