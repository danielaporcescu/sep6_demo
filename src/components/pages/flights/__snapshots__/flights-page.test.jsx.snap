// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly FlightsPage 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    justify="space-around"
    spacing={2}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={4}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The mean airtime of each of the origins in a table
        </Styled(MuiBox)>
        <MeanAirTime
          data-testid="meanAirTime"
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The top-10 destinations and how many flights were made to these
        </Styled(MuiBox)>
        <Top10DestinationsTable
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          Mean departure and arrival delay for each origin in a table
        </Styled(MuiBox)>
        <MeanDepArrDelay
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={8}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <TotalFlightsPerMonth
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <TotalFlightsPerMonthFromOrigins
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <TotalFlightsPetMonthFromOriginStacked
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <TotalFlightsPetMonthFromOriginPercentage
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <TopTenDestinationsPerOrigin
          isLoaded={false}
        />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
